libscript/src/script-execute.cpp

```diff
@@ -202,10 +209,32 @@ class MCScriptForeignInvocation
             break;
         
         case kMCScriptForeignHandlerLanguageC:
```

Comment:
I fixed this in the MCJavaCheckSignature - it throws a binding error if there is a variadic parameter present.